src/utils/helpers/index.jsx

```diff
@@ -0,0 +1,22 @@
+export const getDate = () => {
+  const date = new Date().toISOString().slice(0, 10).split('-').reverse().join('.');
+  return date;
+};
+
+export const getUid = () => {
+  return Date.now();
+};
+export const sanitize = (text) => {
+  return text.trim().replaceAll(/<\/?[^>]+(>|$)/gi, '');
+};
+export const validate = (text) => {
+  if (text) {
+    const sanitized = sanitize(text);
+    if (sanitized === '') return { status: 'error', message: 'Please enter a valid description' };
+    return { status: 'ok', text: sanitized };
+  } else
+    return {
+      status: 'error',
+      message: 'can not be empty'
+    };
+};
```

Comment:
IMHO, there is no need for else here, as you are early returning !
```suggestion
  } 
  return {
      status: 'error',
      message: 'can not be empty'
   ;
};
```

Reply:
else removed. thanks